### PR TITLE
#160 fix

### PR DESCRIPTION
**Pull request by TikhomirovSergey**
Change list:
- Enum SwipeElementDirection is redesigned. The ability to perform the swiping was with offset was added.
- The swiping using offset from element borders was added to MobileElement.

### CODE REVIEW

**Comment by TikhomirovSergey** [comment]
#160

File: src/main/java/io/appium/java_client/TouchableElement.java [hunk 1]
```diff
@@ -1,52 +1,76 @@
...
+      * @param direction UP, DOWN, LEFT, RIGHT
+      * 
+      * @param duration amount of time in milliseconds for the entire swipe action to
+		 *            take
+      */
+	  public void swipe(SwipeElementDirection direction, int duration);	  
+	  
+	  
+	  /**
+	   * Convenience method for swiping on the given element to the given direction
```
Comment (Jonahss) [inline src/main/java/io/appium/java_client/TouchableElement.java]: What's the difference between offset1 and offset2? Do you mean to say that offset1 is offset from the element that swipe starts on, and offset2 is from the element that the swipe ends on?

Comment (TikhomirovSergey) [inline src/main/java/io/appium/java_client/TouchableElement.java]: offset1 and offset2 are offsets from borders of the element.  
offset1 is the offset from the border where the swiping should be started. 
offset2 is the offset from the border where the swiping should be ended.
If parameter names are not clear how can I rename them?

**Comment by TikhomirovSergey** [comment]
Ok. I am going to rename __offset1__ and __offset2__ to __offsetFromStartBorder__ and __offsetFromEndBorder__. Is it ok?

**Comment by Fanch-** [comment]
For me it's clear.

**Comment by TikhomirovSergey** [comment]
Ok.

**Comment by TikhomirovSergey** [comment]
I've improved parameter names and comments.

**Comment by Jonahss** [comment]
Great :)

**Pull request merged**